api/query.py

```diff
@@ -47,6 +47,20 @@ class Location(NamedTuple):
     stop: Position
 
 
+class Type(NamedTuple):
+    location: Dict[str, Position]
+    annotation: str
+
+    def extract_function_model(self) -> str:
+        results = re.findall("\(.*?\)", self.annotation)
+        return results[0] if results else ""
```

Comment:
I would suggest an exception, rather than an empty string. If someone called this method, they thought the represented type was a function. If that assumption isn't true, we shouldn't hide that fact